lib/Controller/SettingsController.php

```diff
@@ -88,4 +94,24 @@ public function deprovision(int $id): JSONResponse {
 
 		return new JSONResponse([]);
 	}
+
+	/**
+	 *
+	 * @return JSONResponse
+	 */
+	public function setAntiSpamEmail(array $data): JSONResponse {
```

Comment:
```suggestion
	public function setAntiSpamEmail(string $spam, string $ham): JSONResponse {
```

you should be able to spread to type-hinted arguments